.travis.yml

```diff
@@ -21,6 +21,7 @@ matrix:
 install:
   - git clone --depth 1 git://github.com/astropy/ci-helpers.git
   - source ci-helpers/travis/setup_conda.sh
+  - conda install -c conda-forge zarr pyspectral
```

Comment:
pyspectral is already installed via the environment variables at the top of this file. Zarr should be added to the conda dependencies.

Reply:
Done!